torch/onnx/_internal/fx/custom_operator.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+import torch
+from .function_dispatcher import _ATENLIB_FUNCTIONS, _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
+
+ONNX_CUSTOM_OP_LIB = torch.library.Library("onnx_custom", "DEF")
```

Comment:
A link/short intro to `torch.library.Library` could be very helpful. Helps with understanding what later call of `.impl` and `.define` means.